src/components/modal/modal-add-edit-content.tsx

```diff
@@ -43,169 +60,158 @@ const ModalAddEditContent = ({
   const [releaseDate, setReleasedate] = useState(
```

Comment:
You don't need all this `useState`.
You don't need to handle forms state manually via `useState`, formik makes this for you.

Reply:
done